frontend/public/components/dashboard/project-dashboard/getting-started/DeveloperFeaturesGettingStartedCard.tsx

```diff
@@ -29,17 +29,17 @@ export const DeveloperFeaturesGettingStartedCard: React.FC = () => {
   if (isHelmEnabled && isHelmVisible && !disabledAddActions?.includes('helm')) {
     links.push({
       id: 'helm-charts',
-      title: t('devconsole~Discover certified Helm Charts'),
+      title: t('public~Helm chatbot AI sample'),
```

Comment:
should this title maintain the verb at the start of the sentence? e.g., 

```suggestion
      title: t('public~Try the sample AI Chatbot Helm chart'),
```